runtime/doc/eval.txt

```diff
@@ -1060,7 +1060,8 @@ Floating point numbers can be written in two forms:
 	[-+]{N}.{M}[eE][-+]{exp}
 
 {N} and {M} are numbers.  Both {N} and {M} must be present and can only
-contain digits.
+contain digits, except that in |Vim9| script in {N} single quotes between
+digits are ignored.
```

Comment:
I'm not sure that's strictly true; we've used the commit ID in the past for (obvious) cherry-picks (and not gotten yelled at for it ;)). Just don't use the standard "update runtime files" description -- instead, be more specific -- and it should be unambiguous.

But (only) @janlazo is the authority here.

Reply:
Is it OK to use `vim-patch:hash:` for partial ports? (and not something less-specific like `runtime/hash: details`?)

I would have expected the former to only be used for full ports like with regular patches. 👀

Reply:
That's how we have used it, and I would argue that, yes, since keeping the link to the runtime update commit is most important. (So it will show up in `git blame` for the cherry-picked file, and when searching for that commit in the log -- that you get multiple hits then is not a bad thing.)

Reply:
(Patches are a different thing, since they are atomic -- we either port them (as far as applicable) or not (yet).)

Reply:
(And, yes, that should be documented somewhere...)

Reply:
That makes sense. I was going to update the Wiki, but came across https://github.com/neovim/neovim/pull/14419#issuecomment-824474227, which says `vim-patch:` can only be used for full RT ports, but interestingly in janlazo's huge RT in #14424 it seems there were some partial ports that did use the token, but explained what was left out in the commit message. :shrug:

In other words I think I'll leave the documentation to someone else 😅

Reply:
Yeah, but that commit a) didn't make clear what was cherry-picked and b) used the "update runtime files" message reserved for full ports.

Reply:
Seems good. I'll get around to mentioning this on the Wiki.

Reply:
> but came across #14419 (comment), which says vim-patch: can only be used for full RT ports

This is exactly what confused me the most.